src/App.js

```diff
@@ -4,26 +4,75 @@ import './App.css'
 
 import LandingPage from './components/LandingPage'
 import LogInPage from './components/LogInPage'
+import CreateAccountPage from './components/CreateAccountPage'
+import UserHome from './components/UserHome'
 
 export default class App extends Component {
   constructor (props) {
     super(props)
     this.state = {
-      accountName: ''
+      createAccount: '',
+      loginAccount: '',
+      activeAccount: '',
+      activeAccountInfo: {},
+      dbAccounts: {}
+
     }
-    this.onAccountNameInput = this.onAccountNameInput.bind(this)
+
+    this.onLoginAccountNameInput = this.onLoginAccountNameInput.bind(this)
+    this.onCreateAccountNameInput = this.onCreateAccountNameInput.bind(this)
+    this.updateDBAccounts = this.updateDBAccounts.bind(this)
+    this.updateActiveAccount = this.updateActiveAccount.bind(this)
+    this.updateActiveAccountInfo = this.updateActiveAccountInfo.bind(this)
+  }
+
+  updateDBAccounts (json) {
```

Comment:
When doing camelCasing, I and most devs I have worked with prefer to leave acronyms (like `DB`) un-capitalized, so this would be `updateDbAccounts`